dll/win32/shell32/CDefView.cpp

```diff
@@ -628,19 +628,27 @@ void CDefView::UpdateStatusbar()
 BOOL CDefView::CreateList()
 {
     HRESULT hr;
-    DWORD dwStyle, dwExStyle;
+    DWORD dwStyle, dwExStyle, ListExStyle;
     UINT ViewMode;
 
     TRACE("%p\n", this);
 
     dwStyle = WS_TABSTOP | WS_VISIBLE | WS_CHILDWINDOW | WS_CLIPSIBLINGS | WS_CLIPCHILDREN |
-              LVS_SHAREIMAGELISTS | LVS_EDITLABELS | LVS_AUTOARRANGE;
+              LVS_SHAREIMAGELISTS | LVS_EDITLABELS | LVS_AUTOARRANGE; // FIXME: Why is LVS_AUTOARRANGE here?
```

Comment:
If you remove the LVS_AUTOARRANGE from here, does the shell behave a bit more "as expected" regarding icons placement? (as default behaviour)